onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -2306,11 +2306,23 @@ def aten_upsample_bilinear2d(
         result = _aten_upsample_bilinear2d_output_size(self, output_size)
     else:
         assert scales_h is not None
-        assert scales_h == scales_w
+        assert scales_h == scales_w, f"scale_h({scales_h}) != scale_w({scales_w})"
```

Comment:
I don't understand why there is this restriction.

Reply:
@xiaowuhu could you help?